drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/JunitRunnerHelper.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.scenariosimulation.backend.server.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.guvnor.common.services.shared.test.Failure;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.Runner;
+import org.junit.runner.notification.RunListener;
+
+public class JunitRunnerHelper {
+
+    public static Result runWithJunit(Runner runner, List<Failure> failures) {
+        JUnitCore jUnitCore = new JUnitCore();
+
+        jUnitCore.addListener(new RunListener() {
+            @Override
+            public void testAssumptionFailure(org.junit.runner.notification.Failure failure) {
+                failures.add(failureToFailure(failure));
+            }
+        });
```

Comment:
To be honest I'm not able to verify if it is needed or not so I decided to keep same approach of `Scenario test` asset

Reply:
I see this is not used in tests anywhere. The question is, @danielezonca are you able to create a test which would actually use the overrided method? Because if yes, I vote for adding it and if not I think we can omit this code.